### ISSUE TITLE
Show local files at the top

### ISSUE THREAD

**Issue by seahindeniz** [Stremio/stremio-bugs#580]
**Describe the bug**
The local files are not showing at the top of the list anymore. I have to scroll down to the bottom or have to select local files from the source filter every time I switch between episodes. This is so frustrating issue considering that the previous was showing the local files at the top.

I tried reinstalling the app, didn't work.

**To Reproduce**
Here's a screen record of the issue

https://github.com/Stremio/stremio-bugs/assets/5789670/865461d8-6e4d-44d8-8294-9a8ccdedcee0

**Expected behavior**
I expect local files to be listed at the top of the source list

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 11

![image](https://github.com/Stremio/stremio-bugs/assets/5789670/1cd10ddd-7baa-4ef7-bf71-f7524e7d9a82)

**Additional context**

**Comment by TRtomasz** [comment]
Order is determined by the order in which addons where installed. There was no change in this also since there is addon selector on the top i will close this since this is not a bug.

**Comment by seahindeniz** [comment]
Hi @TRtomasz thank you for detailing. I'm using the app for more than 1 year and I can confirm that it was working before as I expected. I'm not sure if it's possible to manage the installation order. If it was indeed related to the installation order, I believe the local files addon was getting installed first, before. This is no longer the case anymore

**Comment by seahindeniz** [comment]
I just tried downloading the older version to see if it's working, no luck. I couldn't even get the old UI back.

The old version link
https://dl.strem.io/shell-win/v4.4.137/Stremio+4.4.137.exe
And here's a task link that contains a screen record of the previous version of UI that was working as quite expected
https://github.com/Stremio/stremio-bugs/issues/296

**Comment by seahindeniz** [comment]
The only solution that I found is to remove the removable add-ons from Stremio and then install them back. This was a long shot, but it worked. In case if someone troubles with this issue, I can confirm that it's working, FYI @TRtomasz.
Before:
![image](https://github.com/Stremio/stremio-bugs/assets/5789670/23002baa-4a2c-4895-8e60-614f8314acfe)
Now:
![image](https://github.com/Stremio/stremio-bugs/assets/5789670/433e76b3-0cfb-48f5-bd67-a6457a046a48)

However, couple setbacks that frightened me that I couldn't find Torrentio and ThePirateBay+ add-ons from the add-on lists. So I searched and found them on Google.
Here are those links
https://torrentio.strem.fun/
https://thepiratebay-plus.strem.fun/

Although, I'm not sure if these are a valid source, however seems like Stremio is accepting them without any warning, so I think they are alright.
